Makefile.am

```diff
@@ -7,6 +7,7 @@ EXTRA_DIST = pi-blaster.service
 sbin_PROGRAMS = pi-blaster
 pi_blaster_SOURCES = pi-blaster.c mailbox.c
 
+if !DEBIAN_PKG
```

Comment:
Changing `init_d_SCRIPTS` looks pretty strange. Besides backward-incompatibility, the init scripts in `/etc/init.d` should be anything like `foobar` or `foobar.sh` but `pi-blaster.init`. The file named `foo-bar.init` is only used when packaging. It looks OK if only installed manually from source.

And if the name really should be changed, it would be great if there are some solid suggestions (like removing current symlink, renaming the symlink or making a duplication of init script, etc).